packages/clis/parser/types/restructure.d.ts

```diff
@@ -53,7 +53,7 @@ declare module 'restructure' {
     size(): number;
   }
 
-  export class Enum<T extends readonly string[]> extends Base<
+  export class Enum<T extends readonly unknown[]> extends Base<
```

Comment:
Gonna revert this change... never needed to make use of it (and `unknown` is too broad a type).
```suggestion
  export class Enum<T extends readonly string[]> extends Base<
```